scribejava-apis/src/main/java/com/github/scribejava/apis/service/DiscordOAuthServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package com.github.scribejava.apis.service;
+
+import com.github.scribejava.core.builder.api.DefaultApi20;
+import com.github.scribejava.core.model.AbstractRequest;
+import com.github.scribejava.core.model.OAuth2AccessToken;
+import com.github.scribejava.core.model.OAuthConfig;
+import com.github.scribejava.core.model.OAuthConstants;
+import com.github.scribejava.core.oauth.OAuth20Service;
+
+public class DiscordOAuthServiceImpl extends OAuth20Service {
+    public DiscordOAuthServiceImpl(DefaultApi20 api, OAuthConfig config) {
+        super(api, config);
+    }
+
+    @Override
+    public void signRequest(OAuth2AccessToken accessToken, AbstractRequest request) {
+        request.addHeader(OAuthConstants.HEADER, accessToken.getTokenType() + " " + accessToken.getAccessToken());
```

Comment:
" " -> ' ' #microoptimization ;-)

There are a lot of 'Bearers' in the ScribeJava already. I will try to put this to the core, OAuth20Service.java. And then merge your Pull Request.

And yep, I will need credentials. I will return near the 10th January to this bearer-refactoring and your Pull Request. So I will need test credentials near that date.

ps.Thanks! Good, clean Pull Request.

pps.You can add info about new API to the /changelog and /Readme.md